main/webapp/modules/core/scripts/views/data-table/cell-ui.js

```diff
@@ -360,9 +362,25 @@ DataTableCellUI.prototype._searchForMatch = function(suggestOptions) {
   elmts.newButton.html($.i18n('core-buttons/new-topic'));
   elmts.clearButton.html($.i18n('core-buttons/dont-reconcile'));
   elmts.cancelButton.html($.i18n('core-buttons/cancel'));
+  
+	if (!defaultChoice) {
+		elmts.radioSimilar[0].setAttribute("checked", false);
+		elmts.radioOne[0].setAttribute("checked", true);
+	}
 
   var level = DialogSystem.showDialog(frame);
   var dismiss = function() {
+	  if (elmts.radioSimilar[0].checked) {
+		  defaultChoice = true;
+		  elmts.radioSimilar[0].setAttribute("checked", true);
+		  elmts.radioOne[0].setAttribute("checked", false);
```

Comment:
It is not clear to me why the two lines above are needed - why do you need to update the DOM at this stage? (similarly below)

Reply:
Actually I think its not needed since the users choice has been stored in the global variable and the DOM should be updated using the global variable value when the reconciliation match dialog is opened. 
Let me work on it an see the outcome